tests/common/fixtures/advanced_reboot.py

```diff
@@ -127,6 +127,7 @@ def __extractTestParam(self):
         self.postRebootCheckScript = self.request.config.getoption("--post_reboot_check_script")
         self.bgpV4V6TimeDiff = self.request.config.getoption("--bgp_v4_v6_time_diff")
         self.new_docker_image = self.request.config.getoption("--new_docker_image")
+        self.neighborType = self.request.config.getoption("--neighbor_type")
```

Comment:
This likely needs to be added to `tests/platform_tests/args/advanced_reboot_args.py` file as well. How did it work w/o that change?

Reply:
This argument is a common argument defined at a higher level; it's used by macsec tests.